kustomize/overlays/dev/registry.yaml

```diff
@@ -16,19 +16,13 @@ spec:
     spec:
       containers:
         - name: ttl-registry
-          image: registry:2
-          command: ["/entrypoint.sh"]
+          image: ttl-registry
```

Comment:
where does this image come from?

Reply:
okteto image rewrites

https://github.com/replicatedhq/ttl.sh/pull/152/files#diff-dd8070292ce806a326a3d0543b136ba3b5f7930f40d33475780462fdcece2b70R16